src/workerd/jsg/jsg.h

```diff
@@ -303,6 +303,36 @@ class JsExceptionThrown: public std::exception {
 //
 //     JSG_METHOD_NAMED(delete, delete_);
 
+#define JSG_METHOD_UNBOUND(name) \
```

Comment:
I don't feel strongly one way or the other so whatever works I suppose. Given that this pattern will be used extensively in the Node.js compat model I just want something that is easiest long term and `JSG_METHOD_UNBOUND` is quite simple. All the options (`JSG_METHOD_UNBOUND`, a property that returns `cfjs::Function`, a `JSG_STRUCT`) are just trading one complexity for another. Here I simply did the easiest thing that works.

> If that works I'd rather do that, to avoid adding more complexity to JSG.

Which? You mention two alternative approaches. It's not clear which you prefer.

Reply:
Actually, I'm going to remove these from this PR entirely.